### ISSUE TITLE
Spurious 'Statement has empty block' on blade files

### ISSUE THREAD

**Issue by HenkPoley** [kalessil/phpinspectionsea#1448]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended)|
| Language level | PHP 7.2                       |

Current behaviour
--
EA Extended willl give the warning '[EA] Statement has empty block' on the following default Laravel welcome.blade.php code segment, which doesn't look quite empty to me:

```php
                @if (Route::has('register'))
                    <a href="{{ route('register') }}">Register</a>
                @endif
```

Expected behaviour
--
No warning.

The current warning might be because blade doesn't use `then` in `@if` statements.

Environment details
---
PhpStorm 2019.2.5
EA Extended 4.0.1

**Comment by kalessil** [comment]
Fixed!
